indico/modules/events/registration/controllers/management/invitations.py

```diff
@@ -183,7 +183,7 @@ def _process(self):
         skip_moderation = form.skip_moderation.data if 'skip_moderation' in form else False
         skip_access_check = form.skip_access_check.data
         skip_existing = form.skip_existing.data
-        delimiter = form.delimiter.data.char()
+        delimiter = form.delimiter.enum[form.delimiter.data].delimiter
```

Comment:
I'm surprised this was necessary...

Reply:
`form.delimiter.data` was directly the enum name, not the actual enum object. Yes, I was also surprised.

Reply:
Found the problem - `strip_whitespace` thinking string enum members are just strings, and thus stripping them, which of course converts them into a regular string...

Reply:
I see... nice catch!

Reply:
One of the few cases where using an actual debugger was useful (breakpoint at the end of `process_formdata` of the `_EnumFieldMixin` where the data was still correct, and right after stepping out of it the filters got applied)